### ISSUE TITLE
Classes in markdown file are overwritten

### ISSUE THREAD

**Issue by StefanoTesla** [nuxt/content#982]
I have last version of nuxt, nuxt/content and node.js

What is expected??

By default I have h1,h2,h3... with taliwind text-center and I put some table in the markdown tables as this:

```
<div class=" text-center">
  <p class="text-red-500 text-4xl">Arduino Mega</p>
  <ul>
    <li class="py-10 text-yellow-500 text-3xl">5€</li>
    <li class="pt-4">16KHz</li>
    <li class="py-4">8K Memoria</li>
    <li class="py-4">256K Flash</li>
    <li class="py-4">4K EEPROM</li>
    <li class="py-4">3x16bit Timer</li>
    <li class="py-4">13KHz Step Rate</li> 
  </ul>
</div>
```

And after generate the site to static I will see what I see in development

What append?

After generate all pages of nuxt content don't have the style I wanted.
Olso table are not as expected.

tailwind.config.js:

```
const colors = require('tailwindcss/colors')


module.exports = {
  purge: {
    content: [
      './content/**/*.md'
    ],
    options: {
      safelist: ['m_ard', 'top_ard', 'title_ard', 'b_nav_ard', 'card_ard','code_area_ard', 'comment_ard']
    },
  },
    prefix: '',
    important: false,
    separator: ':',
    theme: {
        colors: {
            gray: colors.trueGray,
            red: colors.red,
            yellow: colors.yellow,
            green: colors.green,
            blue: colors.sky,
            orange :colors.orange
          },

    },
    variants: 
    {},
    plugins: [
    ],
  }
```

**Comment by NozomuIkuta** [comment]
@StefanoTesla 

Thank you for providing the reproduction.
I checked your repo, and found the root cause and solution.

**TLDR**

You can fix the issue by setting `purge.content` option in `tailwind.config.js` for PurgeCSS to analyze markdown files in `content` directory.

```javascript
const path = require('path')

module.exports = {
  purge: {
    content: [
      path.join(__dirname, './content/**/*.md')
    ]
  }
}
```

**Details**

The reason why styles in the page are broken in generated pages (i.e. in production) is because the CSS classes used in markdown files are purged on build time by PurgeCSS, as you guessed.

You have to set the `purge.content` option in `tailwind.config.js` for PurgeCSS to analyze markdown files.

With `@nuxtjs/tailwindcss`, only `*.(vue|js|ts)` files in Nuxt default directories (e.g. `pages`, `components`) are specified by default ([link](https://github.com/nuxt-community/tailwindcss-module/blob/55bdd8dbe407378517f4348d342c7e516ec92aa9/src/tailwind.config.ts)).

This is why styles are shown as expected when you define the table as a Vue component.

By the way, it's natural that `content` directory is not specified by default, because Nuxt Content module and Nuxt TailwindCSS module are independent from each other.

**Comment by StefanoTesla** [comment]
> @StefanoTesla
> 
> Thank you for providing the reproduction. I checked your repo, and found the root cause and solution.
> 
> **TLDR**
> 
> You can fix the issue by setting `purge.content` option in `tailwind.config.js` for PurgeCSS to analyze markdown files in `content` directory.
> 
> ```js
> const path = require('path')
> 
> module.exports = {
>   purge: {
>     content: [
>       path.join(__dirname, './content/**/*.md')
>     ]
>   }
> }
> ```
> 
> **Details**
> 
> The reason why styles in the page are broken in generated pages (i.e. in production) is because the CSS classes used in markdown files are purged on build time by PurgeCSS, as you guessed.
> 
> You have to set the `purge.content` option in `tailwind.config.js` for PurgeCSS to analyze markdown files.
> 
> With `@nuxtjs/tailwindcss`, only `*.(vue|js|ts)` files in Nuxt default directories (e.g. `pages`, `components`) are specified by default ([link](https://github.com/nuxt-community/tailwindcss-module/blob/55bdd8dbe407378517f4348d342c7e516ec92aa9/src/tailwind.config.ts)).
> 
> This is why styles are shown as expected when you define the table as a Vue component.
> 
> By the way, it's natural that `content` directory is not specified by default, because Nuxt Content module and Nuxt TailwindCSS module are independent from each other.

I already did this in my project, but not in the demo repo

**Comment by NozomuIkuta** [comment]
@StefanoTesla

Do you mean the issue is not fixed in your project?

Then, there might be some conditions specific to your project.

Without any more information or reproduction, I can't say anything that may helpful to you.
